ui/v2.5/src/index.scss

```diff
@@ -406,32 +406,96 @@ div.dropdown-menu {
   margin: 0 10px;
 }
 
+.rating-20 {
+  background: #f00;
+}
+
+.rating-19 {
+  background: #ff2409;
+}
+
+.rating-18 {
+  background: #ff4812;
+}
+
+.rating-17 {
+  background: #ff6a07;
+}
+
+.rating-16 {
+  background: #ff8000;
+}
+
+.rating-15 {
+  background: #fa8804;
+}
+
+.rating-14 {
+  background: #f39409;
+}
+
+.rating-13 {
+  background: #eca00e;
+}
+
+.rating-12 {
+  background: #e7a811;
+}
+
+.rating-11 {
+  background: #dfb617;
+}
+
+.rating-10 {
+  background: #d2ca20;
+}
+
+.rating-9 {
+  background: #cbb526;
+}
+
+.rating-8 {
+  background: #c39f2b;
+}
+
+.rating-7 {
+  background: #bd8e2f;
+}
+
+.rating-6 {
+  background: #b47435;
+}
+
 .rating-5 {
-  background: #ff2f39;
+  background: #af7944;
 }
 
 .rating-4 {
-  background: $red1;
+  background: #a7805b;
 }
 
 .rating-3 {
-  background: $orange1;
+  background: #a48363;
 }
 
 .rating-2 {
-  background: $sepia1;
+  background: #9e8974;
 }
 
 .rating-1 {
-  background: $dark-gray5;
+  background: #9b8c7d;
+}
```

Comment:
Can we change this so that the new rating systems use classes `rating100-x` and leave the 5-star system to use `rating-x`, and leave the `rating-x` styles unchanged from the existing styles? This way, people on the 5-star system will have the same styling, and they won't need to change their custom css unless they change systems.

Reply:
updated